src/globals.css

```
@@ -0,0 +1,296 @@
+p, li { text-wrap: pretty; }
+
+img { outline: 1px solid rgba(255, 255, 255, 0.1); outline-offset: -1px; }
+.light img { outline-color: rgba(0, 0, 0, 0.1); }
+
+/* Lucide icons: use stroke-width 1.5 site-wide for elegance. */
+svg.lucide { stroke-width: 1.5 !important; }
+
+/* Tabular nums utility (animated numbers must use this). */
+.tabular-nums { font-variant-numeric: tabular-nums; }
+
+@theme inline {
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)